codeworld-api/codeworld-api.cabal

```diff
@@ -27,6 +27,8 @@ Library
                        CodeWorld.Driver
                        CodeWorld.CollaborationUI
   Build-depends:       base                 >= 4.8 && < 5,
+                       bytestring,
+                       codeworld-error-sanitizer,
```

Comment:
This should be either here or in the impl(ghcjs) section, not both.  (Although, see the other comment about rearranging dependencies)